### ISSUE TITLE
Listbox-Button: allow ability to add prefix text to the button.

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1320]
# Feature Request

## Description

Allow ability to add prefix text to the button. This prefix would remain in place every time a new value is chosen from the listbox.

For example:

`<button>Reason: Cannot contact seller</button>`

`prefixText="Reason"`

## Context

We often see teams trying to use listbox-button like a `<select>`, by providing a `<label>` tag. However, this is not how buttons are labelled. A buttons label is derived from its inner text/value (although we *could* also allow it to be constructed from an external text element plus the value, using aria-labelledby, see: #1321).

Let's clear up this confusion and make it easier for devs to correctly label this component and thus cut down a11y defects.

**Comment by agliga** [comment]
This has been published in `6.1.0` and in 7.2.0` beta for now (will go out in `7.2.0`)
